mmv1/third_party/terraform/services/container/resource_container_node_pool_test.go.tmpl

```diff
@@ -4013,15 +4009,15 @@ resource "google_container_cluster" "cluster" {
   deletion_protection = false
   network             = "%s"
   subnetwork          = "%s"
-  min_master_version  = "1.32.0-gke.1448000"
+  min_master_version  = "1.32.1-gke.1357001"
 }
 
 resource "google_container_node_pool" "np1" {
   name               = "%s"
   location           = "us-central1-a"
   cluster            = google_container_cluster.cluster.name
   initial_node_count = 2
-  version            = "1.32.0-gke.1448000"
+  version            = "1.32.1-gke.1357001"
```

Comment:
I don't know if they're testing something specific with an explicit version here? normally, setting the node pool version to the same version as the control plane will Just Work that way anyway without being configured? I took a look at some of the PRs mentioned in the comments here, but still couldn't figure out what is going on.

Reply:
I don't think they are testing something specific.